book_source/developers_guide/Using-Git.Rmd

```diff
@@ -1,78 +1,49 @@
+          [LearnGitBranching](http://pcottle.github.com/learnGitBranching/)
+          [TryGit](http://try.github.com)
 
-For additional tutorials and background see [References](Using-Git.md#references)
+    2. Start with the one-time set-up instructions under [Before any work is done](#before-any-work-is-done). 
+
+    3. To make changes to PEcAn code, follow our [workflow](#workflow-to-make-changes). 
+    
+        - For asking questions, reporting bugs, and requesting features, see our documentation for [reporting issues on GitHub][Overview]
+    
+**Using GitHub** The easiest way to get working with GitHub is by installing the GitHub
+client. For instructions for your specific OS and download of the
+GitHub client, see https://help.github.com/articles/set-up-git.
+This will help you set up an SSH key to push code back to GitHub. To
+check out a project you do not need to have an ssh key and you can use
+the https or git url to check out the code.
 
 **URLs** In the rest of the document will use specific URL’s to clone the code.
 There a few URL’s you can use to clone a project, using https, ssh and
 git. You can use either https or git to clone a repository and write to
-it. The git protocol is read-only.
-
-### PEcAnProject on GitHub: https://github.com/organizations/PecanProject
-
-* PEcAn source code: 
- * https://github.com/PecanProject/pecan.git
- * git@github.com:PecanProject/pecan.git
-* BETYdb source code:
- * https://github.com/PecanProject/bety.git
- * git@github.com:PecanProject/bety.git
-* These instructions apply to other repositories too.
-
-### Milestones, Issues, Tasks
+it. The git protocol is read-only. More information can be found [here](https://help.github.com/articles/which-remote-url-should-i-use/)
 
-The Milestones, issues, and tasks can be used to organize specific features or research projects. In general, there is a heirarchy:
-
-* milestones (Big picture, "Epic"): contains many issues
-* issues (Specific features / bugs, "Story"): may contain a list of tasks; represent 
-* task list (to do list, "Tasks"): list of steps required to close an issue, e.g.:
-```
- * [ ] first do this
- * [ ] then this
- * [ ] completed when x and y
-```
-
-### Quick and Easy
-
-The **easiest** approach is to use GitHub's browser based workflow. This is useful when your change is a few lines, if you are editing a wiki, or if the edit is trivial (and won't break the code). The [GitHub documentation is here](https://help.github.com/articles/github-flow-in-the-browser) but it is simple: finding the page or file you want to edit, click "edit" and then the GitHub web application will automatically forking and branch, then allow you to submit a pull request. However, it should be noted that unless you are a member of the PEcAn project that the "edit" button will not be active and you'll want to follow the workflow described below for forking and then submitting a pull request.
```

Comment:
I've added the basic workflow  and quick and easy section. I deleted the "quick and easy" section because the "quick and easy" directions only apply to people who are members of the project on GitHub and  they should already be using github the regular way. In the end I agree it's good to have all the ways of using git so people can see different methods.